Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -476,6 +476,21 @@ extension TabTrayController: UIScrollViewAccessibilityDelegate {
     }
 }
 
+// There seems to be a bug with UIKit where when the UICollectionView changes it's contentSize 
+// from > frame.size to <= frame.size, the contentSet animation doesn't properly happen and 'jumps' to the
+// final state. This workaround forces the contentSize to always be larger than the frame size so the animation happens more 
+// smoothly. This also makes the tabs be able to 'bounce' when there are not enough to fill the screen which I 
+// think is fine but if needed we can disable user scrolling in this case - :sleroux
+private class TabTrayCollectionViewLayout: UICollectionViewFlowLayout {
+    private override func collectionViewContentSize() -> CGSize {
+        var calculatedSize = super.collectionViewContentSize()
+        if calculatedSize.height < collectionView!.bounds.size.height {
```

Comment:
do we want to be force unwrapping the collectionView at this point? I know it _should_ be there but it feels a little risky.